internal/serve/httphandler/disbursement_handler_test.go

```diff
@@ -1257,12 +1253,32 @@ func Test_DisbursementHandler_PatchDisbursementStatus(t *testing.T) {
 		Email: "email@email.com",
 	}
 	require.NotNil(t, user)
+
 	authManagerMock := &auth.AuthManagerMock{}
+	hMock := &horizonclient.MockClient{}
+	distributionPubKey := "ABC"
+	asset := data.GetAssetFixture(t, ctx, dbConnectionPool, data.FixtureAssetUSDC)
+
+	hMock.On(
+		"AccountDetail", horizonclient.AccountRequest{AccountID: distributionPubKey},
+	).Return(horizon.Account{
+		Balances: []horizon.Balance{
+			{
+				Balance: "10000",
+				Asset: base.Asset{
+					Code:   asset.Code,
+					Issuer: asset.Issuer,
+				},
+			},
+		},
+	}, nil)
```

Comment:
For all mocks, we usually indicate with `Once()`, `Twice()` or `Times(n)` how many times we expect that mocked function to be called. Can you make sure to do that here as well?